kernels/volk/volk_32fc_index_max_32u.h

```diff
@@ -445,76 +375,29 @@ volk_32fc_index_max_32u_u_avx2(uint32_t* target, lv_32fc_t* src0,
     xmm8 = _mm256_add_epi32(xmm8, xmm10);
   }
 
-  xmm10 = _mm256_set1_epi32(4);
-  if (num_bytes >> 5 & 1) {
-    xmm1 = _mm256_loadu_ps((float*)src0);
-
-    xmm1 = _mm256_mul_ps(xmm1, xmm1);
-
-    src0 += 4;
-
-    xmm1 = _mm256_hadd_ps(xmm1, xmm1);
-
-    xmm3 = _mm256_max_ps(xmm1, xmm3);
-
-    xmm4.float_vec = _mm256_cmp_ps(xmm1, xmm3, _CMP_LT_OS);
-    xmm5.float_vec = _mm256_cmp_ps(xmm1, xmm3, _CMP_EQ_OQ);
-
-    xmm11 = _mm256_and_si256(xmm8, xmm5.int_vec);
-    xmm12 = _mm256_and_si256(xmm9, xmm4.int_vec);
-
-    xmm9 = _mm256_add_epi32(xmm11, xmm12);
+  _mm256_storeu_ps((float*)&(holderf.f), xmm3);
+  _mm256_storeu_si256(&(holderi.int_vec), xmm9);
 
-    xmm8 = _mm256_add_epi32(xmm8, xmm10);
+  for(i = 0; i < 8; i++)
+  {
+    if(holderf.f[i] > max)
+    {
+      index = holderi.i[i];
+      max = holderf.f[i];
+    }
```

Comment:
I'm curious about formatting here. I'd start braces on a new line for functions and at the end of a line for `for` and `if`. I thought that would be the clang-format way.

Reply:
I rebased (to pick up the recent clang-format commit) and ran clang-format on this branch, so the formatting should now be fixed.